docs/xgc.md

```diff
@@ -60,6 +62,26 @@ Options that change the behavior of the Garbage Collector (GC).
 
 : This option sets the scan mode for GC operations that evacuate objects in the heap (scavenge operations (`gencon`) and copy forward operations (`balanced`)) to breadth first mode. The scan mode reflects the method for traversing the object graph and is also known as *Cheney's algorithm*.
 
+### `classUnloadingKickoffThreshold`
+
+         -Xgc:classUnloadingKickoffThreshold=<value>
```

Comment:
The option were reversed. The description below fits classUnloadingThreshold, and vice versa.